sample-apps/payment-customization-hide/extensions/hide-first-payment/src/main.rs

```diff
@@ -8,16 +8,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     eprintln!("Hello World");
 
     // read from stdin and write to stdout
-    let payload: Payload = serde_json::from_reader(std::io::BufReader::new(std::io::stdin()))?;
+    let input: Input = serde_json::from_reader(std::io::BufReader::new(std::io::stdin()))?;
     let mut out = std::io::stdout();
     let mut serializer = serde_json::Serializer::new(&mut out);
-    function(payload)?.serialize(&mut serializer)?;
+    function(input)?.serialize(&mut serializer)?;
     Ok(())
 }
 
-fn function(payload: Payload) -> Result<FunctionResult, Box<dyn std::error::Error>> {
-    let (input, _config) = (payload.input, payload.configuration);
-    return Ok(build_result(input.payment_methods[0].id.clone()));
+fn function(input: Input) -> Result<FunctionResult, Box<dyn std::error::Error>> {
+    Ok(build_result(input.payment_methods[0].id.clone()))
```

Comment:
Does it make sense to use the metafield's value?

Reply:
I guess just for a static hide first payment script we don't need config, will update!